packages/react-core/src/demos/Tabs.md

```diff
@@ -383,6 +383,7 @@ TabsOpenWithSecondaryTabsDemo = () => {
           <TabContentBody>
             <Tabs
               isSecondary
+              // remove in v5 breaking change release; secondary variant will have border by default
```

Comment:
Can we remove these comments and add a note in #8104
There is no need to expose this to the consumer on the website.